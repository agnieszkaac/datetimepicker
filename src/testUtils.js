export const setDate = (date) => {
  Date.now = jest.fn(() => Date.parse(date));
};

export const setToday = () => {
  const today = "2020-01-01";
  Date.now = jest.fn(() => Date.parse(today));
};
