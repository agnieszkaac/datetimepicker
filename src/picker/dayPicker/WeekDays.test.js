import React from "react";
import { render } from "@testing-library/react";

import { WeekDays } from "./";
import "../../testUtils";

describe("<WeekDays /> should", () => {
  it("match snapshot", () => {
    const { asFragment } = render(<WeekDays />);
    expect(asFragment()).toMatchSnapshot();
  });
});
