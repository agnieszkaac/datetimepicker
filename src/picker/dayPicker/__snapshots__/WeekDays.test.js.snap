// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WeekDays /> should match snapshot 1`] = `
<DocumentFragment>
  <span
    class="day-of-week"
  >
    Sun
  </span>
  <span
    class="day-of-week"
  >
    Mon
  </span>
  <span
    class="day-of-week"
  >
    Tue
  </span>
  <span
    class="day-of-week"
  >
    Wed
  </span>
  <span
    class="day-of-week"
  >
    Thu
  </span>
  <span
    class="day-of-week"
  >
    Fri
  </span>
  <span
    class="day-of-week"
  >
    Sat
  </span>
</DocumentFragment>
`;
